signupsignin.js

```diff
@@ -0,0 +1,130 @@
+    var emailVerified = user.emailVerified;
+    var photoURL = user.photoURL;
+    var isAnonymous = user.isAnonymous;
+    var uid = user.uid;
+    var providerData = user.providerData;
+  };
+});
+
+function signedInPage() {
+  document.body.innerHTML = `
+  <h1 class="regTitle"><img src="logo1.png" class="gtlogo" alt="goodteens"></h1>
+  <h1 class="regTitle"> Create your profile </h1>
+  <form name="RegForm" action="process.php" method="post" autocomplete="off">
+      <div class="row">
+          <div class="col-sm-3">
+          </div>
+          <div class="col-sm-3">
+              <label for="firstname">First name</label>
+              <input type="text" size=65 name="firstname" class="form-control" id="firstname" placeholder="albert"
+                  required>
+          </div>
+          <div class="col-sm-3">
+              <label for="city">City</label>
+              <input type="text" size=40 name="city" class="form-control" id="city" placeholder="munich" required>
+          </div>
+          <div class="col-sm-3">
+          </div>
+      </div>
+      <div class="row">
+          <div class="col-sm-3">
+          </div>
+          <div class="col-sm-3">
+              <label for="lastname">Last name</label>
+              <input type="text" size=65 name="lastname" class="form-control" id="lastname" placeholder="einstein"
+                  required>
+          </div>
+          <div class="col-sm-3 autocomplete">
+          <label for="validationCustom05">Country</label>
+          <input class="form-control" id="country" type="text" name="country" placeholder="Country" required>
+      </div>
+          <div class="col-sm-3">
+          </div>
+      </div>
+      <div class="row">
+          <div class="col-sm-3">
+          </div>
+
+          <div class="col-sm-3">
+              <label for="agecheck">Your age in years</label>
+              <input type="number" size=65 name="yearuser" class="form-control" id="agecheck"
+                  placeholder="type number" required>
+          </div>
+          <div class="col-sm-3">
+                  <button class="btn" type="submit">Create an account</button></div>
+          </div>
+          <div class="col-sm-3">
+          </div>
+      </div>
+  </form>
+  <script src="registration.js"></script>
+  `};
+
+const provider = new firebase.auth.GoogleAuthProvider();
+
+firebase.auth().signInWithPopup(provider).then(function (result) {
+  // This gives you a Google Access Token. You can use it to access the Google API.
+  var token = result.credential.accessToken;
+  // The signed-in user info.
+  var user = result.user;
+  // ...
+}).catch(function (error) {
+  // Handle Errors here.
+  var errorCode = error.code;
+  var errorMessage = error.message;
+  // The email of the user's account used.
+  var email = error.email;
+  // The firebase.auth.AuthCredential type that was used.
+  var credential = error.credential;
+  // ...
+});
+
+signUpEl.addEventListener(click, signUp);
```

Comment:
first argument of `addEventListener` is a string but right now it's a variable `click` and browser will argue about it.
```js
  signUpEl.addEventListener('click', signUp);
```

Reply:
oh yes, thanks.